e2e/integration/filters/showMoreAndLess.spec.js

```diff
@@ -31,9 +31,9 @@ describe('Show more/less facet filters',() => {
 
       cy.get('@subjectAccordion')
         .find('button[data-cy=show-more-about]')
-        .should('contain.text', 'Show more (746)')
+        .should('contain.text', 'Show more (90)')
         .click()
-        .should('contain.text', 'Show more (736)');
+        .should('contain.text', 'Show more (80)');
```

Comment:
This seems to be related with Steel's changes to Algolia's configuration

Reply:
@fdalcin & @richard015ar I broke this test with an unintended configuration change (max values per facet should be 1000, not 100)
![Screenshot from 2021-06-23 12-17-57](https://user-images.githubusercontent.com/13485451/123155520-0fd9bc80-d41d-11eb-9fad-b875c0e8eac7.png)

I've changed it back to the desired value in all the Algolia indices, so you can discard this change and go back to the old test (show more 746, 736, etc, instead of 90, 80)

Reply:
Reverted to old test values